adapters/mqtt-vertx-base/src/main/java/org/eclipse/hono/adapter/mqtt/impl/HttpBasedMessageMapping.java

```diff
@@ -120,7 +120,8 @@ private void mapMessageRequest(
             }
         });
         headers.add(MessageHelper.APP_PROPERTY_ORIG_ADDRESS, ctx.message().topicName());
-        headers.add(HttpHeaders.CONTENT_TYPE.toString(), ctx.contentType());
+        headers.add(HttpHeaders.CONTENT_TYPE.toString(),
```

Comment:
FMPOV we should not set the header at all if the device did not provide a content-type. Maybe the mapper wants to handle this case specifically?